wiki/Submission/Beatmap_Ranking_Procedure/en.md

```diff
@@ -1,44 +1,26 @@
 
-When a beatmap is first made, it is in the Pending or Work in Progress sections of the forum. These maps are referred to as "unranked". Unranked maps do not have a scoreboard and go to the Graveyard section of the forum after a period of inactivity. In order for a map to have a scoreboard and to be moved to the "Ranked" section of the forums, a map must go through the beatmap approval process. First it must get bubbled, and then ranked. Only BNs can bubble and subsequently rank a beatmap.
-
-Bubble ![Thinking icon](/wiki/shared/icon/bubble.gif "Thinking icon")
+Nominations 
 --------------------------------------------------------------------
 
-Before a beatmap can become ranked by a [BN](/wiki/People/Beatmap_Nomination_Group) (Beatmap Nominator), it must first get bubbled. In order for a [beatmap](/wiki/Beatmaps) to become bubbled, it must follow and be in accordance with the [ranking criteria](/wiki/Ranking_Criteria), and it must have at least 12 [Star Priority](/wiki/Glossary/#star-priority) (or SP). You can actively gain SP in two different ways. First, you can have others help mod your map. If you find their mod to be useful and give kudosu, you will gain 1 SP. You and/or others can also spend [kudosu](/wiki/Glossary/#kudosu) in order to "star" your map (A star costs 1 kudosu, and can be done through either the [mapper](/wiki/Glossary/#beatmapper)'s profile or through the [map](/wiki/Beatmaps)'s thread). Once a beatmap has at least 12 SP, a BN can bubble it if they feel it meets two conditions:
-
-1.  The beatmap meets the [Ranking Criteria](/wiki/Ranking_Criteria). Basically, this must mean that the map has the correct [timing](/wiki/Beatmap_Editor/Timing), has at least 2 [difficulties](/wiki/Difficulties) (one of which is an [Easy](/wiki/Difficulties/osu!/Easy) or a [Normal](/wiki/Difficulties/osu!/Normal)), and is overall both mapped to the music and playable.
-2.  The beatmap meets overall quality standards that are not part of the Ranking Criteria. For example, if a slider is not very pretty, then a BN would likely request for it to be made prettier before bubbling it.
+In order for a [beatmapset](/wiki/Glossary/#beatmapset) to receive a nomination, it must follow and be in accordance with the [ranking criteria](/wiki/Ranking_Criteria) and must have at least 5 hypes. Hypes can be opbtained by having other users hype your beatmap in the Discussion page of the map, or under the map's description. Once a beatmap has at least 5 hypes, a BN can nominate it if they feel it meets two conditions:
```

Comment:
should move or add these links to the intro post

Reply:
done, thanks for the suggestion